examples/batch_convert.py

```diff
@@ -30,9 +30,18 @@ def export_documents(
             with (output_dir / f"{doc_filename}.json").open("w") as fp:
                 fp.write(json.dumps(conv_res.render_as_dict()))
 
+            # Export Text format:
+            with (output_dir / f"{doc_filename}.txt").open("w") as fp:
+                fp.write(conv_res.render_as_text())
+
             # Export Markdown format:
             with (output_dir / f"{doc_filename}.md").open("w") as fp:
                 fp.write(conv_res.render_as_markdown())
+                
+            # Export Document Tages format:
+            with (output_dir / f"{doc_filename}.md").open("w") as fp:
```

Comment:
this should have a different name. what about `f"{doc_filename}.doctags"`

Reply:
yup, fixed